### ISSUE TITLE
How to display navigation bar only on specific screens?

### ISSUE THREAD

**Issue by tomkur** [aksonov/react-native-router-flux#116]
Having `hideNavBar` on `Router` seems to make it impossible or difficult to show navigation bar only on specific screens. I'm thinking: don't show navigation bar on login screen, but show on other screens, after logging in.

How do I achieve that?

**Comment by aksonov** [comment]
Try to use hideNavBar for Route.

**Comment by tomkur** [comment]
Thanks for answering, but I'm afraid it doesn't work (I'm using version 2.0.5). Is `hideNavBar` really supported by `Route`? Can't find anything in the docs.

**Comment by aksonov** [comment]
It is in testing phase, so it is not in docs yet.

**Comment by tomkur** [comment]
I see, is it supposed to work in 2.0.5 or only master?

**Comment by aksonov** [comment]
2.0.5

**Comment by tomkur** [comment]
I'll share my code. Neither of these two examples works for me:
```
<Router hideNavBar={true}>
    <Route name='login' initial={true} component={Login}/>
    <Route name='dashboard' component={Dashboard} title='Dashboard' hideNavBar={false}/>
</Router>
```

```
<Router>
    <Route name='login' initial={true} component={Login} hideNavBar={true}/>
    <Route name='dashboard' component={Dashboard} title='Dashboard' hideNavBar={false}/>
</Router>
```

**Comment by aksonov** [comment]
Thanks, looks like i didn't committed some changes, will check now.

**Comment by tomkur** [comment]
Looking forward to this, thanks!

**Comment by aksonov** [comment]
Please check latest master branch

**Comment by tomkur** [comment]
For future reference: fix available in 2.0.6

**Comment by tomkur** [comment]
Thanks @aksonov, works like a charm!
